api/sk-serve/db/player_service.go

```diff
@@ -51,3 +54,36 @@ func (p *PlayerImpl) GetPlayerById(ctx context.Context, playerId *string) (*mode
 
 	return player, nil
 }
+
+func (p *PlayerImpl) Create(ctx context.Context, playerInput model.NewPlayerNfl) (*model.PlayerNfl, error) {
+	players := p.Client.Collection(collectionPlayers)
+
+	playerId := generatePlayerId(playerInput.PlayerName)
+
+	newPlayer := model.PlayerNfl{
+		ID:           playerId,
+		PlayerName:   playerInput.PlayerName,
+		TeamNfl:      *playerInput.TeamNfl,
+		Position:     playerInput.Position,
+		PositionRank: *playerInput.PositionRank,
+		Birthday:     *playerInput.Birthday,
+		OverallRank:  *playerInput.OverallRank,
+		Avatar:       "",
+	}
+
+	result, err := players.Doc(newPlayer.ID).Set(ctx, newPlayer)
+	if err != nil {
+		return nil, err
+	}
+
+	fmt.Println(result.UpdateTime.Day())
```

Comment:
Was this for debugging? can we remove it?

Reply:
Yep - I'll remove it.